lib/codecs/src/encoding/format/syslog.rs

```diff
@@ -293,27 +294,35 @@ fn default_nilvalue() -> String {
 }
 
 fn add_log_source(log: &LogEvent, buf: &mut String) {
+    let namespace_name = log.get(event_path!("kubernetes", "namespace_name"));
```

Comment:
Is there a place to expose these as configurable?  I'm thinking of internal refactoring case where these don't "live" off root or the OTLP where they may have other values.  It may require you to do something like this: https://github.com/ViaQ/vector/pull/183/files#diff-bbb203636e7d043378bc1c3daf679a702201d76e9adf40eeb9ce9c06165191ebR71

Reply:
Good idea, we should do this in another CR.